src/plone/restapi/services/workflow/transition.py

```diff
@@ -16,6 +16,7 @@
 from zope.interface import implementer
 from zope.publisher.interfaces import IPublishTraverse
 from zope.publisher.interfaces import NotFound
+from DateTime import DateTime
```

Comment:
Ordering of imports does not conform to the Plone style guide (https://docs.plone.org/develop/styleguide/python.html#grouping-and-sorting). But that was already the case before that change.

Reply:
I will look into this...